src/components/avatar/Avatar.js

```diff
@@ -78,14 +78,14 @@ class Avatar extends PureComponent {
   };
 
   render() {
-    const { className, onClick, selected, size, shape, ...others } = this.props;
+    const { className, selectable, selected, size, shape, ...others } = this.props;
```

Comment:
I think you actually fixed a bug "by accident" 😄. Before this change, the `onClick` prop was actually not put on the component, it was only used for the is-selectable class. 

Now that you replaced it by selectable, it's in the `others` and will be spread onto the box later on.

Reply:
Omg, this is so true! 🙈 😄